### DISCUSSION TITLE
Compiling and Testing Moose

### DISCUSSION THREAD

**Discussion by Oops-Qiao** [idaholab/moose#21499] [Q&A Getting Started] [answered]
I am not able to find the closing button.

**Comment by Oops-Qiao** [comment]


**Reply by GiudGiud** [reply]
what's the output of `mamba list` ?
and the diagnostic script in moose/scripts ?

**Reply by Oops-Qiao** [reply]
![image](https://user-images.githubusercontent.com/70176014/177507644-6dd82fe9-321c-4f04-a310-ce582e0ea5ad.png)

**Reply by milljm** [reply]
It looks like the compilation was terminated by external means.

You may be running out of memory during the build. Seeing those diagnostics would really help (what @GiudGiud asked for). You can create a diagnostic log by performing the following:

```bash
cd moose/scripts
./diagnostics.sh > my_log.log
```
and attaching the file created `my_log.log`.

**Comment by GiudGiud** [comment, marked as answer]
Closing this, interface not letting me select an empty message as the answer
